requirements/software-metadata-requirements.md

```diff
@@ -0,0 +1,187 @@
+# CLARIAH Software Metadata Requirements and Instructions
+
+## Introduction
+
+This document specifies requirements and instructions for software metadata for
+CLARIAH software and services. It is aimed at the developers of software and
+attempts to provide clear documentation.
+
+The main principle underlying the way we keep our software metadata is that
+**metadata is kept at the source and by the source of the software**. This
```

Comment:
‘by the source’ sounds a bit odd. Do you mean by the _maintainers_ of the software?